forge-gui/res/cardsfolder/upcoming/aurelias_vindicator.txt

```diff
@@ -0,0 +1,16 @@
+Name:Aurelia's Vindicator
+ManaCost:2 W W
+Types:Creature Angel
+PT:4/2
+K:Flying
+K:Lifelink
+K:Ward:2
+K:Disguise:X 3 W
+T:Mode$ TurnFaceUp | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ When CARDNAME enters the battlefield, you may exile up to three other target creatures from the battlefield and/or creature cards from graveyards.
+SVar:TrigExile:DB$ ChangeZone | TargetMin$ 0 | TargetMax$ X | IsCurse$ True | ValidTgts$ Creature.Other | TgtPrompt$ Choose up to X other target creatures from the battlefield and/or creature cards from graveyards | RememberChanged$ True | Origin$ Battlefield,Graveyard | Destination$ Exile
```

Comment:
ExiledWith is the future

Reply:
I wasn't sure since it said "the" exiled cards rather than cards exiled with

Reply:
> 406.6. An object may have one ability printed on it that causes one or more cards to be exiled, and another ability that refers either to "the exiled cards" or to cards "exiled with [this object]." These abilities are linked: the second refers only to cards that have been exiled due to the first. See rule 607, "Linked Abilities."